azure/functions/timer.py

```diff
@@ -1,23 +1,30 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
 import json
 import typing
 
 from azure.functions import _abc as azf_abc
-
 from . import meta
 
 
 class TimerRequest(azf_abc.TimerRequest):
 
-    def __init__(self, *, past_due: bool) -> None:
+    def __init__(self, *, past_due: bool, schedulestatus: dict,
+                 schedule: dict) -> None:
         self.__past_due = past_due
+        self.__schedulestatus = schedulestatus
+        self.__schedule = schedule
 
     @property
     def past_due(self):
         return self.__past_due
 
+    @property
+    def schedulestatus(self):
+        return self.__schedulestatus
+
+    @property
+    def schedule(self):
```

Comment:
Can we also write the expected return types for all the properties?

Reply:
I have included expected return types in updated file. Thanks.